packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -474,6 +477,12 @@ export const GiantTrackTile = ({
 
   const goToUploadWithRemix = useCallback(() => {
     if (!trackId) return
+
+    if (!currentUserId) {
```

Comment:
update goToUploadWithRemix to use "useRequiresAccountCallback", handles it all for you